Makefile

```diff
@@ -7,17 +8,26 @@ MONGO_SETTINGS=MONGO_CONNECTION=${MONGO_CONNECTION} \
 	CUSTOMCONNSTR_mongo_collection=${CUSTOMCONNSTR_mongo_collection} \
 	CUSTOMCONNSTR_mongo_settings_collection=${CUSTOMCONNSTR_mongo_settings_collection}
 
-.PHONY: all coverage test travis
+# XXX.bewest: Mocha is an odd process, and since things are being wrapped and
+# transformed, this odd path needs to be used, not the normal wrapper.
+# When ./node_modules/.bin/mocha is used, no coverage information is generated.
+MOCHA=$(shell which mocha ./node_modules/mocha/bin/_mocha | head -n 1)
```

Comment:
I think you can also use `./node_modules/.bin/_mocha` here, if you want to be a little more consistent with the comment and the istanbul path.